test/hotspot/jtreg/runtime/FieldLayout/ArrayBaseOffsets.java

```diff
@@ -1,113 +0,0 @@
-/*
```

Comment:
This file too suffered the same fate; all contents were removed but the file was not deleted.